api/templates/emails/registration_plaintext.html.twig

```diff
@@ -0,0 +1,8 @@
+Hello {{ user.firstname }},
```

Comment:
That doesn't generate HTML.

Reply:
Do i need to generate HTML for the PlainText tab ?